README.md

```diff
@@ -1,7 +1,10 @@
 /usr/etc and /etc/\*/\*.d check
 ==============================
 
-File check result: &nbsp; [![Build Status](https://travis-ci.com/yast/usr-etc-test.svg?branch=master)](https://travis-ci.com/yast/usr-etc-test)
+[![Workflow Status](https://github.com/yast/usr-etc-test/workflows/CI/badge.svg?branch=master)](
+https://github.com/yast/usr-etc-test/actions?query=workflow%3ACI+branch%3Amaster)
+[![Check](https://github.com/yast/usr-etc-test/workflows/Check/badge.svg?branch=master)](
```

Comment:
I would probably more highlight that this is the most important result. Probably move it to top and use better name.